packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1867,13 +1917,25 @@ abstract class Element implements BuildContext {
     }
   }
 
+  /// Remove [renderObject] from the render tree.
+  ///
+  /// The default implementation of this function simply walks down the tree and
+  /// recursively calls [detachRenderObject]. The
+  /// [RenderObjectElement.detachRenderObject] override does the actual work of
+  /// removing [renderObject] from the render tree.
```

Comment:
"walks down the tree" implies to me that it does a walk of more than one level, but I guess it's technically correct, you don't say it does it more than once. Still, maybe this could be clarified a modicum.